packages/x-components/src/components/modals/__tests__/base-modal.spec.ts

```diff
@@ -87,13 +93,57 @@ describe('testing Base Modal  component', () => {
 
     expect(wrapper.find(getDataTestSelector('default-slot-overridden')).exists()).toBe(true);
   });
+
+  it('changes the focus to the correct element when the modal opens', async () => {
+    let { wrapper, setOpen, appendToBody } = mountBaseModal({
+      // eslint-disable-next-line max-len
+      defaultSlot: `<div><button data-test="expected-focus">First button</button><button>Second button</button></div>`,
```

Comment:
...a bit silly suggestion, but for newbies like me, it would be faster to read this way..., up to you if you want to change this everywhere 🙊 🙊 
```suggestion
      defaultSlot: `<div>
                               <button data-test="expected-focus">First button</button>
                               <button>Second button</button>
                            </div>`,
```